build.gradle

```diff
@@ -13,8 +13,10 @@ subprojects {
   apply plugin: 'nebula.netflixoss'
   apply plugin: 'checkstyle'
 
-  sourceCompatibility = 1.8
-  targetCompatibility = 1.8
+  tasks.withType(JavaCompile) {
```

Comment:
This change was essentially what our gradle experts suggested. Here's the exact idiom they recommend:

```groovy
subprojects {
  tasks.withType(JavaCompile) {
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8
  }
}
```

Reply:
Modified the code to use `JavaVersion.VERSION_1_8`